docker-entrypoint.sh

```diff
@@ -60,29 +60,29 @@ if expr "$1" : "apache" 1>/dev/null || [ "$1" = "php-fpm" ] || [ "${NEXTCLOUD_UP
             if [ -n "${NEXTCLOUD_ADMIN_USER+x}" ] && [ -n "${NEXTCLOUD_ADMIN_PASSWORD+x}" ]; then
                 # shellcheck disable=SC2016
                 install_options='-n --admin-user "$NEXTCLOUD_ADMIN_USER" --admin-pass "$NEXTCLOUD_ADMIN_PASSWORD"'
-                if [ -n "${NEXTCLOUD_TABLE_PREFIX+x}" ]; then
+                if [ -n "${NEXTCLOUD_TABLE_PREFIX}" ]; then
```

Comment:
An emtpy table prefix is valid. It even is the default if nothing is present (see else clause).

Also this changes do not work for optional parameters if the variable is not set:

```shell
/entrypoint.sh: 63: /entrypoint.sh: NEXTCLOUD_TABLE_PREFIX: parameter not set
```